jedi/api/__init__.py

```diff
@@ -759,10 +759,11 @@ def x():
             global_var = 3
 
             def bar(foo):
-                return foo + 1 + global_var
+                return int(foo + 1 + global_var)
 
             def x(foo):
```

Comment:
Can you remove this foo in the param, it shouldn't be there.

Reply:
I can't understand why.
before refactor:
```
            global_var = 3

            def x():
                foo = 3.1
                x = int(foo + 1 + global_var)
```
after refactor, as show in test, the result is:
```
            global_var = 3

            def bar(foo):
                return int(foo + 1 + global_var)

            def x(foo):
                foo = 3.1
                x = bar(foo)
```

Reply:
No it's not, in your own test you have `def x():` both before and after